lib/Dialect/CGGI/Transforms/ExpandLUT.cpp

```diff
@@ -68,7 +68,7 @@ struct ExpandLUT : impl::ExpandLUTBase<ExpandLUT> {
     alignment::populateWithGenerated(patterns);
     patterns.add<ExpandLutLinComb>(context);
 
-    (void)applyPatternsAndFoldGreedily(getOperation(), std::move(patterns));
+    (void)applyPatternsGreedily(getOperation(), std::move(patterns));
```

Comment:
something like this:
```suggestion
    // TODO (#1221): Investigate whether folding (default: on) can be skipped here.
    (void)applyPatternsGreedily(getOperation(), std::move(patterns));
```